src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+      data-id={todo.id}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
let's handle the case if user is null here, UserInfo goal is to render the user info, that component doesn't need to know that the user can be null